app/controllers/predicates/InFlightReturnFrequencyPredicate.scala

```diff
@@ -35,9 +35,11 @@ class InFlightReturnFrequencyPredicate @Inject()(customerCircumstancesService: C
                                                  val serviceErrorHandler: ServiceErrorHandler,
                                                  val messagesApi: MessagesApi,
                                                  implicit val appConfig: AppConfig,
-                                                 implicit val ec: ExecutionContext)
+                                                 val mcc: MessagesControllerComponents)
   extends ActionRefiner[User, User] with I18nSupport {
 
+  override implicit protected val executionContext: ExecutionContext = mcc.executionContext
```

Comment:
Doesn't need to override, can just be `implicit val executionContext: ExecutionContext = mcc.executionContext`